src/core/sock/sock-app.cpp

```diff
@@ -175,7 +180,6 @@ int app_conf::proc_envoy(int __op, int __fd)
         } else if (__op == EPOLL_CTL_DEL) {
             std::lock_guard<decltype(g_p_app->m_lock)> lock(g_p_app->m_lock);
             g_p_app->map_listen_fd.erase(__fd);
-            g_p_app->map_thread_id.erase(gettid());
```

Comment:
[question] Does it mean that the collection is cleared only at exit? Since the worker_id is stable for a thread, why this deletion was implemented initially?

Reply:
> [question] Does it mean that the collection is cleared only at exit? Since the worker_id is stable for a thread, why this deletion was implemented initially?

worker_id is stable for thread. worker reinitialization is not supported.